.gitignore

```diff
@@ -15,6 +15,7 @@ eshell/lastdir
 my-org/
 org-files/
 semanticdb/
+srecode-map.el
```

Comment:
I've also had this file pop up in my `.emacs.d` recently. I just deleted it before git committing but it keeps coming back, it should be added to .gitignore.